specs/electra/beacon-chain.md

```diff
@@ -332,7 +344,7 @@ class ExecutionPayload(Container):
     transactions: List[Transaction, MAX_TRANSACTIONS_PER_PAYLOAD]
     withdrawals: List[Withdrawal, MAX_WITHDRAWALS_PER_PAYLOAD]
     blob_gas_used: uint64
-    excess_blob_gas: uint64
+    excess_blob_gas: uint64 # [Deprecated in Electra: compute blob gas in CL]
```

Comment:
I believe this field can be repurposed and renamed to `base_fee_per_blob_gas`, this is to make EL block self-contained in terms of execution which is required for optimistic sync (in an edge case but still). Then, this parameter must be verified in the `process_execution_payload`:
```python
assert payload.base_fee_per_blob_gas == get_base_fee_per_blob_gas(state)
```